tests/integration/run.bats

```diff
@@ -167,6 +167,12 @@ function teardown() {
 	[ "$status" -eq 0 ]
 	[[ "$output" = *"Hello World"* ]]
 	[[ "$output" = *"runc-dmz: using /proc/self/exe clone"* ]]
+	if [ "$EUID" -eq 0 ] && is_kernel_gte 5.1; then
```

Comment:
Guess we also need to add something like `&& grep -qFw overlay /proc/filesystems` here.